src/main/java/org/soniakbew/daos/ClientDao.java

```diff
@@ -0,0 +1,52 @@
+package org.soniakbew.daos;
+
+import org.soniakbew.models.Client;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClientDao {
+    public List<Client> getAllClients() throws SQLException {
+        List<Client> clients = new ArrayList<>();
+
+        try (Connection connection = DatabaseConnector.getConnection()) {
+            Statement statement = connection.createStatement();
+
+            ResultSet resultSet = statement.executeQuery(
+                    "SELECT clientId, name FROM `client`;"
+            );
+
+            while (resultSet.next()) {
+                Client client = new Client(
+                        resultSet.getInt("clientId"),
+                        resultSet.getString("name")
```

Comment:
consider abstracting this out to a mapper for both gets

Reply:
will be abstracted out to a mapper in the service class

Reply:
if its abstracted in the service class why is it in here?